openrtb2/content.go

```diff
@@ -106,39 +110,42 @@ type Content struct {
 	//   URL of the content, for buy-side contextualization or review.
 	URL string `json:"url,omitempty"`
 
+	// Attribute:
+	//   cattax
+	// Type:
+	//   integer
+	// Description:
+	//   The taxonomy in use. Refer to the AdCOM list List: Category
+	//   Taxonomies for values.
+	CatTax adcom1.CategoryTaxonomy `json:"cattax,omitempty"`
+
 	// Attribute:
 	//   cat
 	// Type:
 	//   string array
 	// Description:
-	//   Array of IAB content categories that describe the content
-	//   producer. Refer to List 5.1.
+	//   Array of IAB content categories that describe the content.
+	//   The taxonomy to be used is defined by the cattax field. If no
+	//   cattax field is supplied IAB Content Category Taxonomy 1.0 is
+	//   assumed.
 	Cat []string `json:"cat,omitempty"`
 
 	// Attribute:
 	//   prodq
 	// Type:
 	//   integer
 	// Description:
-	//   Production quality. Refer to List 5.13
-	ProdQ *ProductionQuality `json:"prodq,omitempty"`
-
-	// Attribute:
-	//   videoquality
-	// Type:
-	//   integer; DEPRECATED
-	// Description:
-	//   Note: Deprecated in favor of prodq.
-	//   Video quality. Refer to List 5.13.
-	VideoQuality *ProductionQuality `json:"videoquality,omitempty"`
+	//   Production quality. Refer to List: Production Qualities in AdCOM 1.0.
+	ProdQ adcom1.ProductionQuality `json:"prodq,omitempty"`
```

Comment:
The OpenRTB 2.5 enum defines a "0 - unknown" entry whereas AdCOM does not. I don't have a strong opinion how to handle this difference.

Reply:
Why, https://github.com/InteractiveAdvertisingBureau/AdCOM/blob/master/AdCOM%20v1.0%20FINAL.md#list_productionqualities defines 0 Unknown prod quality. All good here.

Reply:
Updated Go to include ProductionUknown